services/core-web/src/styles/generic/layout.scss

```diff
@@ -259,7 +259,7 @@ html {
   height: 2px;
 }
 
-.btn--dropdown {
+.ant-btn.btn--dropdown {
```

Comment:
little increase in importance here- this is for the "expand/collapse" buttons on search filter forms. The jumping in size was caused by the hover rule taking priority but not the general rule. It was annoying me a lot and I thought it may have been contributing to increased render times.